packages/cwd-macros/src/lib.rs

```diff
@@ -355,13 +310,31 @@ pub fn info_query(metadata: TokenStream, input: TokenStream) -> TokenStream {
 /// ```
 #[proc_macro_attribute]
 pub fn proposal_module_query(metadata: TokenStream, input: TokenStream) -> TokenStream {
+    let i = cwd_interface_path("voting::InfoResponse");
+
     merge_variants(
         metadata,
         input,
         quote! {
         enum Right {
+            /// Returns the address of the DAO this module belongs to
             #[returns(::cosmwasm_std::Addr)]
-            Dao {}
+            Dao {},
+            /// Returns contract version info
+            #[returns(#i)]
+            Info { },
+            /// Returns the number of proposals that have been created in this module.
+            #[returns(u64)]
+            ProposalCount {},
+            /// Gets the current proposal creation policy for this module.
+            #[returns(cwd_voting::pre_propose::ProposalCreationPolicy)]
+            ProposalCreationPolicy {},
+            /// Lists all of the consumers of proposal hooks for this module.
+            #[returns(cwd_hooks::HooksResponse)]
+            ProposalHooks {},
+            /// Lists all of the consumers of vote hooks for thismodule.
+            #[returns(cwd_hooks::HooksResponse)]
```

Comment:
```suggestion
            #[returns(::std::primitive::u64)]
            ProposalCount {},
            /// Gets the current proposal creation policy for this module.
            #[returns(::cwd_voting::pre_propose::ProposalCreationPolicy)]
            ProposalCreationPolicy {},
            /// Lists all of the consumers of proposal hooks for this module.
            #[returns(::cwd_hooks::HooksResponse)]
            ProposalHooks {},
            /// Lists all of the consumers of vote hooks for this module.
            #[returns(::cwd_hooks::HooksResponse)]
```

know it's a little verbose, but nice to keep our macros hygienic. unlikely to ever be a problem, but when it is it's pretty hard to debug.